src/mono/wasm/features.md

```diff
@@ -0,0 +1,37 @@
+# Browser or JS engine features
+
+dotnet for wasm can be compiled with various MSBuild flags which enable the use of browser features. If you need to target an older version of the browser, then you may need to disable some of the dotnet features or optimizations.
+
+For full set of MSBuild properties [see top of](src\mono\wasm\build\WasmApp.targets)
+For set of [browser WASM features see](https://webassembly.org/roadmap/)
+
+# Multi-threading
+Is enabled by `<WasmEnableThreads>true</WasmEnableThreads>`.
+It requires HTTP headers similar to `Cross-Origin-Embedder-Policy:require-corp` and `Cross-Origin-Opener-Policy:same-origin`.
+See also https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer#security_requirements
+
+# SIMD - Single instruction, multiple data
+Is performance optimization enabled by default. It requires recent version of browser.
+You can disable it by `<WasmEnableSIMD>false</WasmEnableSIMD><WasmBuildNative>true</WasmBuildNative>`.
+[See also](https://github.com/WebAssembly/simd/blob/master/proposals/simd/SIMD.md)
+
+# EH - Exception handling
+Is performance optimization enabled by default. It requires recent version of browser.
```

Comment:
It might be worth trying to enumerate the most common cutoffs for this here since a lot of users will be confused by 'but I installed Chrome 197 on my iPhone' because Chrome on iOS is a webview

Reply:
I will address it here https://github.com/dotnet/runtime/pull/89469